protos/batch.proto

```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 
 option java_multiple_files = true;
 option java_package = "sawtooth.sdk.protobuf";
+option go_package = "sawtooth_sdk/protobuf";
```

Comment:
Hmm, can you do `git log` locally? I am seeing a discrepancy between the ordering here and local.